### PR TITLE
Create LICENSE

### PR DESCRIPTION
**Pull request by wenqifan03**

### CODE REVIEW

**Comment by golsutsg** [comment]
Hi, Wenqi.
I have tried to reproduce your results with the implementation provided by you hosted on Dropbox. However, it executes too slow even using GPU and performs worse than the results claimed in the paper. So may you share the whole process to reproduce your experiments? Like how do you split the training and test set and hyper-parameters used? And BTW, where should I download datasets mentioned in your paper?

**Comment by xhcgit** [comment]
Why don't you publish the data processing code？

**Comment by xhcgit** [comment]
I think you should review your code, it is too slow, and your code have a very serious memory leak problem.

**Comment by golsutsg** [comment]
> I think you should review your code, it is too slow, and your code have a very serious memory leak problem.

Based on my communication with the author through emails, 

1. He said that his code is based on GraphSage-PyTorch and very straightforward, although time-consuming.
2. He also mentioned that there will be one simple version of GraphRec which is less time-consuming and flexible. And he will release it if possible.

The above points are all from the author himself, just for reference.

**Comment by lcwy220** [comment]
> > I think you should review your code, it is too slow, and your code have a very serious memory leak problem.
> 
> Based on my communication with the author through emails,
> 
> 1. He said that his code is based on GraphSage-PyTorch and very straightforward, although time-consuming.
> 2. He also mentioned that there will be one simple version of GraphRec which is less time-consuming and flexible. And he will release it if possible.
> 
> The above points are all from the author himself, just for reference.



> > I think you should review your code, it is too slow, and your code have a very serious memory leak problem.
> 
> Based on my communication with the author through emails,
> 
> 1. He said that his code is based on GraphSage-PyTorch and very straightforward, although time-consuming.
> 2. He also mentioned that there will be one simple version of GraphRec which is less time-consuming and flexible. And he will release it if possible.
> 
> The above points are all from the author himself, just for reference.

I also see another pytorch implement and I run that pyrotch version and got much worse results compared with the claimed results in the paper. So I wonder what about your results? Can you share with me your results?

**Comment by lcwy220** [comment]
@golsutsg I also see another pytorch implement and I run that pyrotch version and got much worse results compared with the claimed results in the paper. So I wonder what about your results? Can you share with me your results?

**Comment by yukeqi12345** [comment]
What exactly does the data represent?

**Comment by happylee** [comment]
> > I think you should review your code, it is too slow, and your code have a very serious memory leak problem.
> 
> Based on my communication with the author through emails,
> 
> 1. He said that his code is based on GraphSage-PyTorch and very straightforward, although time-consuming.
> 2. He also mentioned that there will be one simple version of GraphRec which is less time-consuming and flexible. And he will release it if possible.
> 
> The above points are all from the author himself, just for reference.

Hello, have you reached the code of the paper?
Now I have been unable to reach the result of the paper, which is 0.04 less

**Comment by zengwei-007** [comment]
@golsutsg Can you share the data processing code?Thank you!

**Comment by zengwei-007** [comment]
> > > I think you should review your code, it is too slow, and your code have a very serious memory leak problem.
> > 
> > 
> > Based on my communication with the author through emails,
> > 
> > 1. He said that his code is based on GraphSage-PyTorch and very straightforward, although time-consuming.
> > 2. He also mentioned that there will be one simple version of GraphRec which is less time-consuming and flexible. And he will release it if possible.
> > 
> > The above points are all from the author himself, just for reference.
> 
> Hello, have you reached the code of the paper?
> Now I have been unable to reach the result of the paper, which is 0.04 less

Can you share the data processing code?Thank you!

**Comment by xhcgit** [comment]
> > > > I think you should review your code, it is too slow, and your code have a very serious memory leak problem.
> > > 
> > > 
> > > Based on my communication with the author through emails,
> > > 
> > > 1. He said that his code is based on GraphSage-PyTorch and very straightforward, although time-consuming.
> > > 2. He also mentioned that there will be one simple version of GraphRec which is less time-consuming and flexible. And he will release it if possible.
> > > 
> > > The above points are all from the author himself, just for reference.
> > 
> > 
> > Hello, have you reached the code of the paper?
> > Now I have been unable to reach the result of the paper, which is 0.04 less
> 
> Can you share the data processing code?Thank you!

This work is sick, you don't need to follow it.

**Comment by tianhz5** [comment]
> > > I think you should review your code, it is too slow, and your code have a very serious memory leak problem.
> > 
> > 
> > Based on my communication with the author through emails,
> > 
> > 1. He said that his code is based on GraphSage-PyTorch and very straightforward, although time-consuming.
> > 2. He also mentioned that there will be one simple version of GraphRec which is less time-consuming and flexible. And he will release it if possible.
> > 
> > The above points are all from the author himself, just for reference.
> 
> Hello, have you reached the code of the paper?
> Now I have been unable to reach the result of the paper, which is 0.04 less

Would you kindly share your data processing code? I used the modified version of data processing code published in the other pull request but it will cause key error when I run the GraphRec with the social relation part.
